site/app/entities/poll/Option.php

```diff
@@ -18,6 +18,9 @@ class Option {
     #[ORM\GeneratedValue]
     protected int $id;
 
+    #[ORM\Column(name: "author_id", type: Types::TEXT)]
+    protected string $author_id;
```

Comment:
Shouldn't this technically be a nullable string?

Reply:
Making it nullable was necessary to set a foreign key constraint. Done in [2a28f32](https://github.com/Submitty/Submitty/pull/10197/commits/2a28f32959d508c5e133fa3ee93f73c602d309f8)